docs/DeveloperGuide.md

```diff
@@ -6,32 +6,45 @@
 
 ## Design & implementation
```

Comment:
Just a general suggestion, you might want to explain the name of your product because I cannot tell what ARCS mean intuitively.